sparkly/testing.py

```diff
@@ -120,28 +147,132 @@ def tearDown(self):
         for fixture in self.fixtures:
             fixture.teardown_data()
 
-    def assertDataFrameEqual(self, actual_df, expected_data, fields=None, ordered=False):
+    def assertDataFrameEqual(self,
+                             actual_df,
+                             expected_data,
+                             fields=None,
+                             ordered=False,
+                             places=None):
         """Ensure that DataFrame has the right data inside.
 
         Args:
             actual_df (pyspark.sql.DataFrame|list[pyspark.sql.Row]): Dataframe to test data in.
             expected_data (list[dict]): Expected dataframe rows defined as dicts.
             fields (list[str]): Compare only certain fields.
             ordered (bool): Does order of rows matter?
+            places (int|None): Compare floats up to given decimal places.
```

Comment:
Hahaha! Using a greek letter was a strong argument, but I opted to follow NumPy's convention instead (rtol/atol as in https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_allclose.html#numpy.testing.assert_allclose)